dGame/dComponents/SkillComponent.h

```diff
@@ -161,6 +161,14 @@ class SkillComponent : public Component {
      */
     static void HandleUnmanaged(uint32_t behaviorId, LWOOBJID target);
```

Comment:
No need to duplicate the entire function, just change this to `static void HandleUnmanaged(uint32_t behaviorId, LWOOBJID target, LWOOBJID originator = LWOOBJID_EMPTY);`